_data/sidebars/k8smain-sidebar.yml

```
@@ -289,6 +294,11 @@ entries:
       output: web, pdf
       type: page
 
+    - title: Using the OPA Gatekeeper charm
```

```suggestion
    - title: OPA Gatekeeper
```
I think we should shorten this.  The phrase "using the ... charm" doesn't bring much to the table imho.  This is also a bit more consistent with some of our other headings like `[keepalived|HAcluster|MetalLB]`.